### PR TITLE
Remove sbt update from steward job

### PR DESCRIPTION
**Pull request by etspaceman**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -513,10 +513,6 @@ jobs:
           java-version: 11
           cache: sbt
 
-      - name: sbt update
-        if: matrix.java == 'temurin@11' && steps.setup-java-temurin-11.outputs.cache-hit == 'false'
-        run: sbt '++ ${{ matrix.scala }}' reload +update
```
Comment (armanbilge) [inline .github/workflows/ci.yml:518]: So the issue is that it's still trying to cache like sbt. I think we should have a helper to generate a java-setup step with caching disabled. That would remove this parameter and also the `sbt update` step.

Comment (etspaceman) [reply .github/workflows/ci.yml:518]: oh hm, let me look at that

Comment (etspaceman) [reply .github/workflows/ci.yml]: Alright, let's see how you like this version :D

File: github-actions/src/main/scala/org/typelevel/sbt/gha/WorkflowStep.scala [hunk 2]
```diff
@@ -41,16 +41,20 @@ object WorkflowStep {
     UseRef.Public("actions", "checkout", "v3"),
     name = Some("Checkout current branch (fast)"))
 
-  def SetupJava(versions: List[JavaSpec]): List[WorkflowStep] = {
+  def SetupJava(versions: List[JavaSpec], includeSbt: Boolean = true): List[WorkflowStep] = {
```
Comment (armanbilge) [inline github-actions/src/main/scala/org/typelevel/sbt/gha/WorkflowStep.scala:44]: Or maybe `enableSbtCaching` or something

```suggestion
  def SetupJava(versions: List[JavaSpec], enableCaching: Boolean = true): List[WorkflowStep] = {
```

**Review by armanbilge — APPROVED** [review]

**Pull request merged**